docs/middleware/response/raise_error.md

```diff
@@ -14,6 +14,10 @@ response returns with a 4xx or 5xx status code. All exceptions are initialized
 providing the response `status`, `headers`, and `body`.
 
 ```ruby
+conn = Faraday.new(url: 'http://sushi.com') do |faraday|
```

Comment:
We have moved to httpbingo URLs in most everywhere, can we change to that, here, too?

Reply:
https://github.com/lostisland/faraday/pull/1384 example here

Reply:
Yea sure ! :)